BioArchLinux/tbtools/lilac.yaml

```diff
@@ -2,14 +2,10 @@ maintainers:
 - github: kiri2002
   email: kiri@vern.cc
 build_prefix: extra-x86_64
-pre_build_script: |
-    update_pkgver_and_pkgrel(_G.newver.lstrip('v'))
-    run_cmd(['updpkgsums'])
 post_build_script: |
-    git_add_files('PKGBUILD')
-    git_commit()
+  git_add_files('PKGBUILD')
+  git_pkgbuild_commit()
+  git_commit()
```

Comment:
I believe `git_pkgbuild_commit()` equals to:
```
git_add_files('PKGBUILD')
git_commit()
```

Reply:
I am not sure

Reply:
Okay, you are right